### PR TITLE
Adjusts King's Laser Against APC/Tank

### PR DESCRIPTION
**Pull request by Wisemonster**
## About The Pull Request
Adjusts the King's laser against the apc/tank to deal damage against their hitboxes. Also reduces the damage from 150 damage per 0.3 seconds to 15 damage per tile per 0.3 seconds
## Why It's Good For The Game
Consistancy and intuitiveness, Also currently King's laser shreds APCs/Tanks. This will make it more forgiving to being briefly caught by the king's laser, while also making it usable on more parts of said vehicles outside of the center.
## Changelog
:cl:
balance: King's laser now deals damage to tanks/APCs based on the tiles hit, 15 damage per tile every 0.3 seconds, down from 150 every 0.3 seconds.
/:cl:

### CODE REVIEW

**Comment by TheLeopold** [comment]
By the way, laser's description says that it can only hurt living beings.
You might want to change that.

**Comment by iforgotmeword** [comment]
that is a pretty big nerf to a very very long cast time power
making 40 per tile per 0.3 i think would be better
or 10 damage per 0.1 sec per tile

**Comment by Wisemonster** [comment]
> that is a pretty big nerf to a very very long cast time power making 40 per tile per 0.3 i think would be better or 10 damage per 0.1 sec per tile

Both of these would increase King's damage against tanks/APCs. Keep in mind, King's current dps against these vehicles is 450. This pr changes that to 135 (180 if it's also hitting the middle)

**Review by Kuro020 — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm [hunk 1]
```diff
@@ -405,9 +405,12 @@
 				human_victim.take_overall_damage(15, BURN, updating_health = TRUE)
 				human_victim.flash_weak_pain()
 				animation_flash_color(human_victim)
-			else if(isvehicle(victim))
+			else if(isvehicle(victim) || ishitbox(victim))
 				var/obj/vehicle/veh_victim = victim
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm:409]: can't typecast as vehicle now

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm (hunk 1 above)
Comment (Wisemonster) [reply code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm:409]: Fixed

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**